docs/kernel/modify_kernel.md

```diff
@@ -2,105 +2,183 @@
 
 ## Tutorial: Modify the Image Kernel
 
-In some situations you may want to build and test variations of the default CBL-Mariner Kernel.  Because the kernel is also a package, the process is similar to adding a new package as discussed in the previous section.  
+In some situations, you may want to build and test variations of the default CBL-Mariner kernel.  Because the `kernel` is also a package, the process is similar to adding a new package as discussed in the previous section.  
 
-To begin, copy the complete contents of the CBL-Mariner kernel spec folder into your clone of the CBL-MarinerTutorials repo.  The following assumes you have already cloned CBL-Mariner and the CBL-MarinerTutorials repo and both are nested under a git folder:
+### Prep your Kernel SPEC Environment
+The following assumes you have already completed the [Prepare your Environment](docs/getting_started/prepare_environment.md) steps. To begin creating a custom kernel, copy the contents of the CBL-Mariner `kernel` spec folder into your clone of the `CBL-MarinerTutorials` repo.
 
 ```bash
-user@machine:~/git$ cp -r CBL-Mariner/SPECS/kernel/ CBL-MarinerTutorials/SPECS/kernel/ 
+#
+# The following assumes CBL-Mariner and CBL-MarinerTutorials were cloned under a folder named `git`
+#
+
+# Copy all the kernel contents to CBL-MarinerTutorials
+user@machine:~/git$ cp -r CBL-Mariner/SPECS/kernel/ CBL-MarinerTutorials/SPECS/kernel/
+#
+# ---or---
+#
+# Copy only the kernel building components to CBL-MarinerTutorials
+user@machine:~/git$ rsync -a  --exclude 'CVE*' CBL-Mariner/SPECS/kernel CBL-MarinerTutorials/SPECS/ 
```

Comment:
Thoughts on limiting the choices and prescribing one option during the tutorial?